src/protagonist/DLCS.Repository/Messaging/AssetNotificationSender.cs

```diff
@@ -82,7 +96,17 @@ public async Task<bool> SendIngestAssetRequest(Asset assetToIngest, Cancellation
         return statusCode;
     }
 
-    public Task SendAssetModifiedNotification(ChangeType changeType, Asset? before, Asset? after)
+    private async Task<bool> SendCleanupAssetRequest(Asset assetToCleanup, CancellationToken cancellationToken = default)
+    {
+        var notificationRequest = new
```

Comment:
Message needs to include Customer `id` and `name` (path-element) as per acceptance criteria

Reply:
I've fixed this in #[b804d5f](https://github.com/dlcs/protagonist/pull/568/commits/b804d5f01951e4c464698c50c473314754c82dad) - I haven't explicitly added a customer id as it get's added in with the path-element so adding it explicitly felt like duplication